infrastructure/sentry-subgraph/src/referee.ts

```diff
@@ -657,13 +657,30 @@ export function handleBulkRewardsClaimed(event: BulkRewardsClaimedEvent): void {
       log.warning("Failed to find poolChallenges in handleBulkRewardsClaimed for challenge " + event.params.challengeId.toHexString() + " and poolAdress: " + event.params.bulkAddress.toHexString() + ", TX: " + event.transaction.hash.toHexString(), [])
       return
     }
-  
+
     poolChallenges.claimKeyCount = event.params.winningKeys;
     poolChallenges.totalClaimedEsXaiAmount = event.params.totalReward;
     poolChallenges.save()
-  
+
     challenge.amountClaimedByClaimers = challenge.amountClaimedByClaimers.plus(event.params.totalReward);
     challenge.save();
+
+    const pool = PoolInfo.load(event.params.bulkAddress.toHexString());
+    if (!pool) {
+      log.warning("Failed to find pool handleBulkRewardsClaimed: pool: " + event.params.bulkAddress.toHexString() + ", TX: " + event.transaction.hash.toHexString(), []);
+      return;
+    }
+    pool.totalAccruedAssertionRewards = pool.totalAccruedAssertionRewards.plus(event.params.totalReward)
+    pool.save()
+
+  } else {
```

Comment:
I think we could remove this else and just return here?

Reply:
I don't think we can, the else handles the case for it beeing a bulkSubmission of a wallet and not a pool (`if (bulkSubmission.isPool)`)